tests/grpc.rs

```diff
@@ -247,6 +249,7 @@ async fn retry_until_bitcoin_funding_info(
         let request = tonic::Request::new(NeedsFundingRequest {
             id: 11,
             blockchain: farcaster::Blockchain::Bitcoin.into(),
+            network_selector: NetworkSelector::LocalNetworks.into(),
```

Comment:
FYI the other tests use `AllNetworks` and here you use `LocalNetworks`

Reply:
Yes, that's intentional to test it.